lib/rspec/core/rake_task.rb

```diff
@@ -122,7 +122,7 @@ def file_inclusion_specification
         if ENV['SPEC']
           FileList[ENV['SPEC']].sort
         elsif String === pattern && !File.exist?(pattern)
-          return if rspec_opts =~ /--pattern/
+          return if [*rspec_opts].any? { |opt| opt =~ /--pattern/ }
```

Comment:
Wondering where `rspec_opts` is coming from. I could only find it being assigned a `nil` [here](https://github.com/rspec/rspec-core/blob/2800fe118fa2efeb87b7bdc7888476373debf6ca/lib/rspec/core/rake_task.rb#L80).

Reply:
It's a writable attribute, so it can be set directly on the task

Reply:
Here's [an example](https://github.com/rspec/rspec-core/blob/master/Rakefile#L22) using an array.